hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/SourceFormatAdapter.java

```diff
@@ -126,4 +127,13 @@ public SourceFormatAdapter(Source source) {
         throw new IllegalArgumentException("Unknown source type (" + source.getSourceType() + ")");
     }
   }
+
+  /**
+   * This method is needed to set Config.DEFAULT_MAX_EVENTS_FROM_KAFKA_SOURCE to some lower value for test cases,
+   * gets called only from TestKafkaSource.java class
+   * @param maxEventsFromKafkaSource
+   */
+  public void setDefaultMaxEventsFromKafkaSource(long maxEventsFromKafkaSource) {
```

Comment:
Added this function, otherwise 5M events will be generated and consumed in test case, which will take a lot of time.

Reply:
Will it take so long if the messages in kafka topic is very small(about 100) while `DEFAULT_MAX_EVENTS_FROM_KAFKA_SOURCE ` is 5M? 
Besides, IMHO, it's a little strange to add this method here, how about resetting `KafkaOffsetGen.Config.DEFAULT_MAX_EVENTS_FROM_KAFKA_SOURCE` directly in `TestKafkaSource` and remove this method?

Reply:
@leesf regarding your first point, basically I wanted to show a scenario where sourceLimit and Config.MAX_EVENTS_FROM_KAFKA_SOURCE_PROP are set to Long.MAX_VALUE and default value comes into play when the number of events in kafka topic is greater than that default value (i.e greater than 5M in normal case). That would require consuming those many events from kafka topic, hence I added the function to reset DEFAULT_MAX_EVENTS_FROM_KAFKA_SOURCE. IMHO, with lesser number of events in topic, this scenario is not depicted properly. WDYT? 

Regarding second point, my bad, I can directly reset the value in TestKafkaSource class. Will remove this function.

Reply:
> @leesf regarding your first point, basically I wanted to show a scenario where sourceLimit and Config.MAX_EVENTS_FROM_KAFKA_SOURCE_PROP are set to Long.MAX_VALUE and default value comes into play when the number of events in kafka topic is greater than that default value (i.e greater than 5M in normal case). That would require consuming those many events from kafka topic, hence I added the function to reset DEFAULT_MAX_EVENTS_FROM_KAFKA_SOURCE. IMHO, with lesser number of events in topic, this scenario is not depicted properly. WDYT?

Make sense.